masschallenge/django-accelerator
# [AC-7044] Clear all Django deprecation warnings on django-accelerator

### Changes introduced in: [AC-7044](https://masschallenge.atlassian.net/browse/AC-7044):
- Resolves `RemovedInDjango20Warning` warning for on_delete
- Resolves `RemovedInDjango20Warning` to use `django.urls` instead of `django.core.urlresolvers`

### How to test

- *From `development` branch*
- Modify the `Makefile` change the `test` command to use `python3 -Wall`
```
test: $(VENV)
	@$(ACTIVATE); DJANGO_SETTINGS_MODULE=settings python3 -Wall $(DJANGO_ADMIN) test $(tests)
```
- Run `make test`
- This should return `RemovedInDjango20Warning` warnings

- *Switch this branch `AC-7044`*
- Run `make test`
- This should return no warnings pertaining to `RemovedInDjango20Warning`
